src/components/UserInfo.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+class UserInfo extends React.Component {
+    formatDate(dateStr){
+        //jan 22 8:00pm
+        let returnStr = "";
+        const dateObj = {
+            "01": "Jan",
+            "02": "Feb",
+            "03": "Mar",
+            "04": "Apr",
+            "05": "May",
+            "06": "Jun",
+            "07": "Jul",
+            "08": "Aug",
+            "09": "Sep",
+            "10": "Oct",
+            "11": "Nov",
+            "12": "Dec"
+        };
+        //format date
+        const listDate = dateStr.split("T");
```

Comment:
let's change all these date functions to use moment.js. Take a look at `QueueList.js` line 38. Also this is docs for moment.js: https://momentjs.com/docs/#/displaying/format/